docker-compose/docker-compose.base.yml

```diff
@@ -24,7 +24,11 @@ services:
     restart: unless-stopped
     environment:
       - JAVA_TOOL_OPTIONS=-Xmx576m #deployment: 768m
-    command: --server.port=80 --spring.config.additional-location=/config/
+    # command: --server.port=80 --spring.config.additional-location=/config/
+    # using host ip address for s3 endpoint because spring cloud aws 2.x doesn't allow
+    # spring.cloud.aws.s3.path-style-access-enabled (only spring cloud aws 3.x), to be revisisted ?
+    # docker-compose doesn't put wildcard dns entries *.minio so we need path style
+    command: --server.port=80 --spring.config.additional-location=/config/ --spring.cloud.aws.s3.path-style-access-enabled=true --cloud.aws.endpoint=http://172.17.0.1:19000 --case-store-bucket=bucket-gridsuite
```

Comment:
I removed --spring.cloud.aws.s3.path-style-access-enabled=true --cloud.aws.endpoint=http://172.17.0.1:19000.
I set in case-server and I use the DNS : https://github.com/powsybl/powsybl-case-server/pull/51